pages/test_analytics/permissions.md

```diff
@@ -43,3 +43,5 @@ From the Member Permissions page, organization admins can toggle whether or not
 * Delete suites
 
 If your organization has teams enabled, the suite creation permissions are managed at a team level. Suite creation permission controls can be found on the Teams Settings page. Without teams enabled, the suite creation permission control can be found on the Member Permissions page.
+
+Note:  The Team-Level permissions on a given pipeline, will override the lower priviledge member permisions in the team settings.
```

Comment:
This page is about permissions in Test Analytics, but I think the customer issue came from Pipelines. This page also links to the Pipelines page for more information: [Controlling user permissions](https://buildkite.com/docs/pipelines/permissions). I think that page is a better spot for this clarification. Perhaps in the [Member permissions](http://localhost:3000/docs/pipelines/permissions#member-permissions) section where it already touches on something similar:

> If your organization has teams enabled, the pipeline creation permissions are managed at a team level. Pipeline creation permission controls can be found on the Teams Settings page. Without teams enabled, the pipeline creation permission control can be found on the Member Permissions page.

What do you think?